fv/bpf_test.go

```diff
@@ -831,6 +831,45 @@ func describeBPFTests(opts ...bpfTestOpt) bool {
 					})
 				})
 
+				Context("with LB test-service configured", func() {
+					var (
+						testSvc          *v1.Service
+						testSvcNamespace string
+					)
+
+					testSvcName := "test-lb-service"
+					tgtPort := 8055
+					srcIPRange := []string{"10.65.0.3/24","10.65.1.2/32"}
+					//var srcIPRange []string
+
+					BeforeEach(func() {
+						testSvc = k8sLBService(testSvcName, "10.101.0.10", w[0][0], 80, tgtPort, testOpts.protocol, srcIPRange)
+						testSvcNamespace = testSvc.ObjectMeta.Namespace
+						_, err := k8sClient.CoreV1().Services(testSvcNamespace).Create(testSvc)
+						Expect(err).NotTo(HaveOccurred())
+						Eventually(k8sGetEpsForServiceFunc(k8sClient, testSvc), "10s").Should(HaveLen(1),
+							"Service endpoints didn't get created? Is controller-manager happy?")
+					})
+                                       It("should have connectivity from all workloads via a service to workload 0", func() {
+                                                ip := testSvc.Spec.ClusterIP
+                                                port := uint16(testSvc.Spec.Ports[0].Port)
+
+                                                cc.ExpectSome(w[0][1], TargetIP(ip), port)
+                                                cc.ExpectSome(w[1][0], TargetIP(ip), port)
+                                                cc.ExpectNone(w[1][1], TargetIP(ip), port)
+                                                cc.CheckConnectivity()
+                                        })
+                                       It("should not have connectivity from external to w[0] via local/remote node", func() {
```

Comment:
Could do with a test where the external node is allowed to access; would be embarrassing if that didn't work for some reason!